WalletWasabi/WabiSabi/Backend/WebClients/MempoolSpaceApiClient.cs

```diff
@@ -0,0 +1,56 @@
+using System.Diagnostics;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using NBitcoin;
+using Nito.AsyncEx;
+using WalletWasabi.Tor.Http.Extensions;
+using WalletWasabi.WabiSabi.Backend.Statistics;
+using WalletWasabi.WabiSabi.Backend.WebClients.Models;
+
+namespace WalletWasabi.WabiSabi.Backend.WebClients;
+
+public class MempoolSpaceApiClient : ITxPropagationVerifier
+{
+	public MempoolSpaceApiClient(Network network, HttpClient httpClient)
+	{
+		httpClient.BaseAddress = new Uri(
+			network == Network.TestNet
+				? "https://mempool.space/testnet/"
+				: "https://mempool.space/");
```

Comment:
Maybe add another constructor with BaseAddress as a third argument? Would allow to easy add checks to other instances too, as mempool.space is open source project, anybody can run an instance of it.

Reply:
😅 I arrived to ask for removal. I just don't get it, why should we have an unused ctor.